TicTacToe/BoardPosition.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicTacToe
+{
+    public class BoardPosition
+    {
+        private Dictionary<int, string> _positions  { get; set; } 
+
+        public BoardPosition()
+        {
+            _positions = new Dictionary<int, string>()
+            {
+                { 1, "1" },
+                { 2, "2" },
+                { 3, "3" },
+                { 4, "4" },
+                { 5, "5" },
+                { 6, "6" },
+                { 7, "7" },
+                { 8, "8" },
+                { 9, "9" },
+            };
+        }
+
+        public string GetChoiceAtPosition(int choiceOneThroughNine)
+        {
+            return _positions[choiceOneThroughNine];
+        }
+
+        public void SetChoiceAtPosition(int choiceOneThroughNine, bool isPlayerOneTurn)
+        {
+            _positions[choiceOneThroughNine] = isPlayerOneTurn ? "X" : "O";
```

Comment:
It would be smart to put code in here to make sure the value is between 1-9 but I felt it would take away from the intent of my refactor.